qiita_pet/support_files/doc/source/tutorials/prepare-templates.rst

```diff
@@ -85,19 +85,31 @@ Required fields for EBI submission
-| ``center_name``                     | free text                                 | Name of the sequencing facility.                                                                                                                                                                                                    |
-+-------------------------------------+-------------------------------------------+-------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------+
-| ``platform``                        | ``Illumina`` or ``FLX`` or ``Titanium``   | The sequencing technology used in the study. ``Illumina`` sequencing data was generated on an Illumina platform; ``FLX`` sequencing data was generated with 454 FLX; ``Titanium`` sequencing data was generated with 454 Titanium   |
-+-------------------------------------+-------------------------------------------+-------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------+
-| ``library_construction_protocol``   | free text                                 | Brief description or reference to the protocol that was used for preparing this amplicon library starting from DNA, usually this includes what genomic region was targeted such as *16S*, *ITS*, *18S*, etc.                        |
-+-------------------------------------+-------------------------------------------+-------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------+
-| ``experiment_design_description``   | free text                                 | High-level description of the study (for example, *A longitudinal study of the gut microbiome of two human subjects*).                                                                                                              |
-+-------------------------------------+-------------------------------------------+-------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------+
++-------------------------------------+-------------------------------------------+--------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------+
```

Comment:
I can't tell from the GitHub interface, but there seems to be something off with the formatting of this table when you build the docs:

![screen shot 2015-10-02 at 3 15 42 pm](https://cloud.githubusercontent.com/assets/375307/10259031/93e72146-6918-11e5-8c3f-2247c49450b3.png)